config/examples/kustomization.yaml

```diff
@@ -18,3 +18,6 @@ resources:
 - vmstaticscrape.yaml
 - vmscrapeconfig.yaml
 - vlogs.yaml
+- vlsingle.yaml
+- vlcluster.yaml
+- vmanomaly.yaml
```

Comment:
As far as I know, olm performs tests for given examples. And there is a problem, it's not possible to run vmanomaly example without License...

Reply:
custom image with anomaly mock in example could solve this issue

Reply:
Could you please add link to this mock image? As far as I see, example doesnt use any mock https://github.com/VictoriaMetrics/operator/blob/master/config/examples/vmanomaly.yaml

Reply:
I suggest to exclude vmanomaly, it's not possible to install it without external secret with license.

Reply:
excluded